From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Fri, 6 Aug 2021 13:12:21 -0400
Subject: [PATCH] Add optional installation of command-line programs.

---
 src/CMakeLists.txt | 46 ++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 46 insertions(+)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 698910b9..d60c0b42 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -397,3 +397,49 @@ install(TARGETS codec2 EXPORT codec2-config
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
     PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/codec2 COMPONENT dev
 )
+
+install(TARGETS
+    c2enc
+    c2dec
+    c2sim
+    cohpsk_ch
+    cohpsk_demod
+    cohpsk_get_test_bits
+    cohpsk_mod
+    cohpsk_put_test_bits
+    deframer
+    fdmdv_channel
+    fdmdv_demod
+    fdmdv_get_test_bits
+    fdmdv_mod
+    fdmdv_put_test_bits
+    fm_demod
+    fmfsk_demod
+    fmfsk_mod
+    framer
+    freedv_data_raw_rx
+    freedv_data_raw_tx
+    freedv_data_rx
+    freedv_data_tx
+    freedv_mixed_rx
+    freedv_mixed_tx
+    freedv_rx
+    freedv_tx
+    fsk_demod
+    fsk_get_test_bits
+    fsk_mod
+    fsk_mod_ext_vco
+    fsk_put_test_bits
+    insert_errors
+    ldpc_dec
+    ldpc_enc
+    ldpc_noise
+    ofdm_demod
+    ofdm_get_test_bits
+    ofdm_mod
+    ofdm_put_test_bits
+    tollr
+    vhf_deframe_c2
+    vhf_frame_c2
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT programs EXCLUDE_FROM_ALL
+)
-- 
2.25.1

