From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Thu, 30 Sep 2021 17:48:33 -0400
Subject: [PATCH] Include <complex.h> in freedv_api.h since it now uses
 _Complex.

---
 src/freedv_api.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/freedv_api.h b/src/freedv_api.h
index 23c5fef8..a70a010a 100644
--- a/src/freedv_api.h
+++ b/src/freedv_api.h
@@ -33,6 +33,7 @@
 #ifndef __FREEDV_API__
 #define __FREEDV_API__
 
+#include <complex.h>
 #include <sys/types.h>
 // This declares a single-precision (float) complex number
 #include "comp.h"
-- 
2.32.0

